metrics/cli/lib/services/firebase/adapter/firebase_cli_service_adapter.dart

```diff
@@ -8,7 +8,7 @@ import 'package:cli/services/firebase/strings/firebase_strings.dart';
 
 /// An adapter for the [FirebaseCli] to implement
 /// the [FirebaseService] interface.
```

Comment:
🏷️ 
```suggestion
/// An adapter for the [FirebaseCli] to implement
/// the [FirebaseService] abstract methods.
```
Please, apply this suggestion to all other adapters.